src/ml-engine/main_app/infrastructure/coin_gecko.py

```
@@ -0,0 +1,209 @@
+        'AO': None,  # Unknown
+        'Obyte': 'byteball',
+        'Zksync': 'zksync',
+        'Mode': 'mode',
+        'Re.al': None,  # Not found
+        'IOTA EVM': 'iota-evm',
+        'Libre': None,  # Not found
+        'Bsquared': None,  # Not found
+        'Rollux': 'rollux'
+    }
+
+    symbol_to_chain = {key: COINGECKO_CHAIN_MAP[value[0].chain] for key, value in pool_data.items() \
```

Ruff (0.11.9)

Line too long (101 > 88)

(E501)